### PR TITLE
PXD-2062 400 Error during SA deletion

### PR DESCRIPTION
**Pull request by benJrohrer**
* check for 400 error, see if SA was actually deleted, raise error otherwise

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
The style in this PR agrees with [`black`](https://github.com/ambv/black). :heavy_check_mark:

This formatting comment was generated automatically by a script in [uc-cdis/wool](https://github.com/uc-cdis/wool).

File: cirrus/google_cloud/manager.py [hunk 1]
```diff
@@ -31,6 +31,9 @@
 from cirrus.google_cloud.services import GoogleAdminService
 from cirrus.google_cloud.utils import get_valid_service_account_id_for_user
 
+from logging import getLogger
```
Comment (Avantol13) [inline cirrus/google_cloud/manager.py]: we should use our logger from https://github.com/uc-cdis/cdislogging

File: cirrus/google_cloud/manager.py [hunk 2]
```diff
@@ -1124,6 +1127,22 @@ def remove_member_from_group(self, member_email, group_id):
...
+                logger.warning(
+                    "When removing {} from group ({}), Google API "
+                    "returned status 400".format(member_email, group_id)
+                )
+                member_emails = [
+                    member.get("email", "")
+                    for member in self.get_group_members(group_id)
+                ]
+                if member_email in member_emails:
+                    raise
```
Comment (Avantol13) [inline cirrus/google_cloud/manager.py]: should log info saying it didn't actually get removed

File: cirrus/google_cloud/manager.py [hunk 3]
```diff
@@ -1124,6 +1127,22 @@ def remove_member_from_group(self, member_email, group_id):
...
+                else:
+                    # reaching this point, indicates the member was succesfully removed
+                    response = {}
```
Comment (Avantol13) [inline cirrus/google_cloud/manager.py]: should log info saying after 400 you got group members from google's api and were able to verify the account was removed

**Pull request merged**